### DISCUSSION TITLE
Hi, I contributed 29$ and I thought it will include social information with out additional pay, how can I achieve that?

### DISCUSSION THREAD

**Discussion by Darda360** [omkarcloud/google-maps-scraper#107] [General]
How can I get also social information using the tool?

**Comment by Chetan11-dev** [comment]
Thank you, @Darda360, for initiating the discussion and for utilizing our Google Maps Scraper Tool. I apologize for any misunderstanding; the tool does not scrape social information by default, as Google Maps typically doesn't display social links on profiles. However, we can indirectly extract this information through their websites. For guidance on scraping social details of profiles, such as email, Facebook, Twitter, etc., you can follow the instructions provided [here](https://github.com/omkarcloud/google-maps-scraper?tab=readme-ov-file#-how-do-i-scrape-social-details-of-profiles-like-email-facebook-twitter-etc). 

Alternatively, if you are not satisfied with tool, you may request a refund and we will be glad to refund you.
